k8s/ithc/common/divorce/div-fps.yaml

```diff
@@ -32,8 +32,6 @@ spec:
       aadIdentityName: divorce
       prometheus:
         enabled: true
-      environment:
```

Comment:
don't we need it for ithc?

Reply:
had a chat with Vivek and John and they said as we don't use them right now then adding this wouldn't add any value at this point of time